apps/web/pages/settings/security/session.tsx

```diff
@@ -0,0 +1,129 @@
+import { GetServerSidePropsContext } from "next";
+import { useState } from "react";
+import { useForm } from "react-hook-form";
+
+import { getLayout } from "@calcom/features/settings/layouts/SettingsLayout";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { userMetadata } from "@calcom/prisma/zod-utils";
+import { trpc } from "@calcom/trpc/react";
+import { Button, Form, Meta, Select, SettingsToggle, showToast } from "@calcom/ui";
+
+import { ssrInit } from "@server/lib/ssr";
+
+const SessionConfigView = () => {
+  const { t } = useLocale();
+  const utils = trpc.useContext();
+  const { data: user } = trpc.viewer.me.useQuery();
+  const mutation = trpc.viewer.updateProfile.useMutation({
+    onSuccess: () => {
+      showToast(t("profile_updated_successfully"), "success");
+      reset(getValues());
```

Comment:
We reset here but also dont reset local `sessionState`

Reply:
On success we know the value for `sessionState` already so it is not set